rocketpy/Flight.py

```diff
@@ -2092,7 +2092,7 @@ def uDotVariableMassNumpyFree(self, t, u, postProcessing=False):
 
         T20 = ((w ^ T00) ^ w) + (w ^ T03) + T04 + weightB + Vector([R1, R2, R3])
 
-        T21 = ((I @ w) ^ w) + T05 @ w + (weightB ^ r_CM) + Vector([M1, M2, M3])
+        T21 = ((I @ w) ^ w) + T05 @ w - (weightB ^ r_CM) + Vector([M1, M2, M3])
```

Comment:
Superb work catching this mistake @MateusStano. Great to see that the results agree now. This linear algebra pull request allow for much more readable formulas in code, on top of that, with this last fix, the results appear to be consistent.